src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -474,14 +472,9 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
         if (isThreeDSecureRequiredError) {
             const clientSecret = error.body.three_ds_result.token;
             let result;
-            let needsConfirm = false;
+            const needsConfirm = false;
```

Comment:
```suggestion
```
line 491 just set **false**


and remove the on **src/payment/payment-methods.mock.ts:637** deleting **reusePaymentIntent** and the parameter **shouldReusePaymentIntent**

Reply:
I'm agree with this change